templates/common/baremetal/files/baremetal-kni-tmpfiles.yaml

```diff
@@ -0,0 +1,8 @@
+mode: 0644
+path: "/etc/tmpfiles.d/kni.conf"
+contents:
+  inline: |
+    D /tmp/nm-system-connections 0755 root root - -
+    D /tmp/nm-system-connections-work 0755 root root - -
```

Comment:
Avoid hardcoded paths in `/tmp` - see <https://cwe.mitre.org/data/definitions/377.html>. A better choice for this is `/run` that only privileged components can write into.

Reply:
And that gets into the fact that IMO really this should be a NetworkManager feature - it should by default look for and parse `/run/NetworkManager/system-connections` too.  We'd just be "backporting" that feature or so.

Did you run this by the NM team?

Reply:
NetworkManager will eventually have the ability for ephemeral connections. That will not come until RHEL 8.3. This is a stop-gap until we get there. But thanks for the advice on /run. I will incorporate that.